my-uw/index.md

```diff
@@ -245,79 +245,77 @@ slug: my-uw
   <h3>Widget Templates</h3>
   <p>Generally, widgets should follow one of the following template structures for consistency and maintainability.</p>
 
-  <h4>List Template</h4>
-  <p>Display a list of up to 5 items. Each list item can have primary text, secondary text, and optional text on the right side. For this template, in the widget creator, choose 'list-of-content'. Here is the sample code:</p>
+  <h4>List of Links Template</h4>
+  <p>To use the list of links template, give the app's entity file a widgetType of 'list-of-links' and provide an object to the widgetConfig with the information for the links. You can provide 1-7 links and the template will display accordingly using the circle-button directive. Sample code for entity file:</p>
   {% highlight html %}
-  <div class="widget-body">
-    <ul class="widget-list">
-      <li>
-        <p class="bold">List item 1<span class="right">More text</span></p>
-        <p>Secondary Text</p>
-      </li>
-      <li>
-        <p class="bold">List item 2<span class="right">...</span></p>
-        <p>Good for a description of the item</p>
-      </li>
-      <li>
-        <p class="bold">List item 3<span class="right">...</span></p>
-        <p>Or for extra information</p>
-      </li>
-      <li>
-        <p class="bold">List item 4<span class="right">...</span></p>
-        <p>...</p>
-      </li>
-      <li>
-        <p class="bold">List item 5<span class="right">...</span></p>
-        <p>...</p>
-      </li>
-    </ul>
-    <a class="btn btn-default launch-app-button">See all</a>
-  </div>
+  <portlet-preference>
+      <name>widgetType</name>
+      <value>list-of-links</value>
+  </portlet-preference>
+  <portlet-preference>
+    <name>widgetConfig</name>
+    <value>{
```

Comment:
The `search-with-links` `widgetConfig` uses `<![CDATA[`; should this `widgetConfig`?